client/src/components/RatingsAndReviews/SortReviews.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react';
+
+const SortReviews = () => {
+  const [value, handleChange] = useState('relevance');
```

Comment:
When using useState, we should use [ value, setValue ] so that it's clear what value 'handleChange' will change. eg.  if the value was zebra we would name it [zebra, setZebra]. Nice work using Hooks!

Reply:
Thanks Bishal! I changed the function name after your comment and pushed it.